### ISSUE TITLE
Logs for harvesting processes

### ISSUE THREAD

**Issue by etj** [geosolutions-it/dati-ckan-docker#5]
`docker logs` does not show logs for harvesting processes.

There are 3 paster processes running:
```
/usr/bin/python2 /usr/bin/paster serve /etc/ckan/default/ckan.ini
/usr/bin/python2 /usr/bin/paster --plugin=ckanext-harvest harvester gather_consumer -c /etc/ckan/default/ckan.ini
/usr/bin/python2 /usr/bin/paster --plugin=ckanext-harvest harvester fetch_consumer -c /etc/ckan/default/ckan.ini
```

Log configuration is at the end of the ckan.ini file, and all the logs are directed to stderr.

`docker logs` only shows logs from the `serve` process and not the logs from the other two.

**Comment by randomorder** [comment]
Cannot find logging config ckan.ini file, can you please point it out?

**Comment by etj** [comment]
```
## Logging configuration
[loggers]
keys = root, ckan, ckanext

[handlers]
keys = console

[formatters]
keys = generic

[logger_root]
level = WARNING
handlers = console

[logger_ckan]
level = INFO
handlers = console
qualname = ckan
propagate = 0

[logger_ckanext]
level = DEBUG
handlers = console
qualname = ckanext
propagate = 0

[handler_console]
class = StreamHandler
args = (sys.stderr,)
level = NOTSET
formatter = generic

[formatter_generic]
format = %(asctime)s %(levelname)-5.5s [%(name)s] %(message)s
```

**Comment by etj** [comment]
Anyway gather and fetch processes are run by `nohup` , so there should be a `nohup.log` file somewhere

**Comment by randomorder** [comment]
We could redirect the output to a log file ( https://stackoverflow.com/questions/4549489/can-i-change-the-name-of-nohup-out )
Might be a good idea to make the path configurable

**Comment by etj** [comment]
May be solved together with #15.

**Comment by randomorder** [comment]
logs available at /var/log/ckan/*

bash-4.2$ tail -f  /var/log/ckan/*
==> /var/log/ckan/fetch_consumer.out <==
2018-02-12 14:03:13,304 INFO  [ckanext.multilang.harvesters.multilang] CSW Multilang harvester: extending ISODocument with PT_FreeText
2018-02-12 14:03:13,720 DEBUG [ckanext.spatial.plugin] Setting up the spatial model
2018-02-12 14:03:13,784 DEBUG [ckanext.spatial.model.package_extent] Spatial tables defined in memory
2018-02-12 14:03:13,796 DEBUG [ckanext.spatial.model.package_extent] Spatial tables already exist
2018-02-12 14:03:13,832 DEBUG [ckanext.harvest.model] Harvest tables defined in memory
2018-02-12 14:03:13,836 DEBUG [ckanext.harvest.model] Harvest tables already exist
2018-02-12 14:03:14,010 DEBUG [ckanext.harvest.queue] Fetch queue consumer registered

==> /var/log/ckan/gather_consumer.out <==
2018-02-12 14:03:13,433 INFO  [ckanext.multilang.harvesters.multilang] CSW Multilang harvester: extending ISODocument with PT_FreeText
2018-02-12 14:03:13,871 DEBUG [ckanext.spatial.plugin] Setting up the spatial model
2018-02-12 14:03:13,935 DEBUG [ckanext.spatial.model.package_extent] Spatial tables defined in memory
2018-02-12 14:03:13,950 DEBUG [ckanext.spatial.model.package_extent] Spatial tables already exist
2018-02-12 14:03:13,969 DEBUG [ckanext.harvest.model] Harvest tables defined in memory
2018-02-12 14:03:13,973 DEBUG [ckanext.harvest.model] Harvest tables already exist
2018-02-12 14:03:14,145 DEBUG [ckanext.harvest.queue] Gather queue consumer registered
